packages/api/lambdas/sf-event-sqs-to-db-records/index.js

```diff
@@ -128,18 +130,26 @@ const handler = async (event) => {
   });
 
   const sqsMessages = get(event, 'Records', []);
+  const batchItemFailures = [];
 
-  return await Promise.all(sqsMessages.map(async (message) => {
+  await Promise.all(sqsMessages.map(async (message) => {
+    let cumulusMessage;
     const executionEvent = parseSQSMessageBody(message);
-    const cumulusMessage = await getCumulusMessageFromExecutionEvent(executionEvent);
-
+    try {
```

Comment:
One last comment on review - in cases where there's a mangled SQS message, we're still sending the entire batch back.     That seems unlikely in most operational situations, but it's a legit corner case.   This is an improvement in either case, so it's not going to hold up the PR, but might be worth considering.